src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
 // write code here
+const employeeList = document.body.querySelector('ul');
+const employees = [...document.body.querySelectorAll('li')];
+
+function convertStringtoNumber(string) {
+  return +(string.replace(/[$,]/gm, ''));
+}
+
+function sortList(list) {
+  list.sort((currentEmployee, nextEmployee) => {
+    return convertStringtoNumber(nextEmployee.dataset.salary)
+    - convertStringtoNumber(currentEmployee.dataset.salary);
+  });
+}
+
+function getEmployees(list) {
+  const arrayOfEmployeesProfiels = list
+    .map(employee => {
+      const employeeProfile = {};
+
+      employeeProfile.name = employee.innerText;
+      employeeProfile.position = employee.dataset.position;
+      employeeProfile.salary = employee.dataset.salary;
+      employeeProfile.age = employee.dataset.age;
+
+      return employeeProfile;
+    });
+
+  return arrayOfEmployeesProfiels;
+}
```

Comment:
```suggestion
function getEmployees(list) {
  return  list.map(employee => {
      const employeeProfile = {};

      employeeProfile.name = employee.innerText;
      employeeProfile.position = employee.dataset.position;
      employeeProfile.salary = employee.dataset.salary;
      employeeProfile.age = employee.dataset.age;

      return employeeProfile;
    });
}
```
arrayOfEmployeesProfiels - плохое имя перменной